README.md

```diff
@@ -612,19 +612,26 @@ Put this into your `$HOME/.zshrc` to call `nvm use` automatically whenever you e
 ```zsh
 # place this after nvm initialization!
 autoload -U add-zsh-hook
+find-old-nvmrc() {
+  local path_
+  path_="${OLDPWD}"
+  while [ "${path_}" != "" ] && [ ! -f "${path_}/.nvmrc" ]; do
+    path_=${path_%/*}
+  done
+  nvm_echo "${path_}"
+}
 load-nvmrc() {
-  local node_version="$(nvm version)"
   local nvmrc_path="$(nvm_find_nvmrc)"
 
   if [ -n "$nvmrc_path" ]; then
     local nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
 
     if [ "$nvmrc_node_version" = "N/A" ]; then
       nvm install
-    elif [ "$nvmrc_node_version" != "$node_version" ]; then
+    elif [ "$nvmrc_node_version" != "$(nvm version)" ]; then
       nvm use
     fi
-  elif [ "$node_version" != "$(nvm version default)" ]; then
+  elif [ -e "$(find-old-nvmrc)/.nvmrc" ] && [ "$(nvm version)" != "$(nvm version default)" ]; then
```

Comment:
which would mean this could be:
```suggestion
  elif [ -e "$(nvm_find_up .nvmrc)" ] && [ "$(nvm version)" != "$(nvm version default)" ]; then
```
?

Reply:
This commit suggestion doesn't work. it searches for `.nvmrc` in the current working, not in the old working directory.